src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -1000,6 +1021,30 @@ describe('findDangerousChanges', () => {
     ]);
   });
 
+  it('should detect interfaces added to interfaces', () => {
+    const oldSchema = buildSchema(`
+      interface OldInterface
+      interface NewInterface
+
+      interface Interface1 implements OldInterface
+    `);
+
+    const newSchema = buildSchema(`
+      interface OldInterface
+      interface NewInterface
+
+      interface Interface1 implements OldInterface & NewInterface
+    `);
+
+    expect(findDangerousChanges(oldSchema, newSchema)).to.deep.equal([
+      {
+        type: DangerousChangeType.INTERFACE_ADDED_TO_OBJECT,
```

Comment:
same here

Reply:
I've renamed this to `IMPLEMENTED_INTERFACE_ADDED` following the other types.